### ISSUE TITLE
Focus ring on Lesson planning resource re-ordering disappears unexpectedly

### ISSUE THREAD

**Issue by lyw07** [learningequality/kolibri#6570]
### Observed behavior

reported by @jeepurs during 0.13.1 beta bug bash:

![2020-02-18 14 58 44](https://user-images.githubusercontent.com/2367265/74785562-6cdcc580-525f-11ea-8ad9-0c4d5b110fb1.gif)

### Expected behavior

users should be able to see the focus ring when reordering resources

### User-facing consequences

slightly confusing what will happen next when using keyboard

### Steps to reproduce

…

### Context

* ubuntu 12.04, chrome 77
* Kolibri version: 0.13.1 beta2 on beta site

**Comment by indirectlylit** [comment]
@jeepurs could you please share your OS & browser for this issue?

**Comment by pcenov** [comment]
@radinamatic this one is still extant in 0.15.1

**Comment by radinamatic** [comment]
@sairina Was this instance covered in the recent testing you did for keyboard navigation?

**Comment by sairina** [comment]
Ooooh, no, I didn't get that one! Add it in!

**Comment by marcellamaki** [comment]
@pcenov @radinamatic - not time sensitive given other Kolibri QA for the patch release, but Amelia and I were looking at this together, and have not been able to reproduce on 15.x on Chrome. @pcenov - can you retest when you have a moment, and if you are able to still replicate it, is it on Chrome, or another browser? Thank you!!

**Comment by pcenov** [comment]
@marcellamaki I'm also not able to replicate this in the latest Kolibri build 0.15.9 so it must have been fixed alongside the other improvements. Thank you as well! :)

**Comment by marcellamaki** [comment]
Closing as no longer replicable
